python_modules/dagster/dagster/utils/__init__.py

```diff
@@ -136,3 +145,15 @@ def get_prop_or_key(elem, key):
 
 def list_pull(alist, key):
     return list(map(lambda elem: get_prop_or_key(elem, key), alist))
+
+
+def get_multiprocessing_context():
```

Comment:
i suspect as we go along we will want a separate file for all of our multiprocessing ish